app/models/intake.rb

```diff
@@ -308,8 +308,6 @@
 #
 #  fk_rails_...  (client_id => clients.id)
 #  fk_rails_...  (matching_previous_year_intake_id => intakes.id)
-#  fk_rails_...  (primary_drivers_license_id => drivers_licenses.id)
```

Comment:
These lines shouldn't be changing in this PR (or at all). Can you merge main & run `rake db:migrate:reset` and see if these changes go away?

Reply:
yes not sure how these snuck in, rebased + removed